charts/elasticsearch/values.yaml

```diff
@@ -1,27 +1,32 @@
+#################################
+#                               #
+#  ElasticSearch Configuration  #
+#                               #
+#################################
 nodeSelector: {}
 affinity: {}
 tolerations: []
 
 images:
   es:
     repository: quay.io/astronomer/ap-elasticsearch
-    tag: 7.8.0
+    tag: 7.10.2
```

Comment:
we need to test bumping elasticsearch, it can break logs in astro-ui.

Reply:
How do we normally test this? Is there an automated way?

Reply:
usually you need to run platform in dev/staging, bump  elasticsearch/kibana images, than create new deployment, push example dags via astro-cli and go to astro-ui and open Logs tab.